beets/config_default.yaml

```diff
@@ -40,14 +40,25 @@ ignore: [".*", "*~", "System Volume Information", "lost+found"]
 ignore_hidden: yes
 
 replace:
+    # Bad chars to _
+    # prohibited in many filesystem paths
+    '[<>:\?\*\|]': _
+    # double quotation mark "
+    "\"": _
+    # \ or /
     '[\\/]': _
+    # starting and closing periods
     '^\.': _
-    '[\x00-\x1f]': _
-    '[<>:"\?\*\|]': _
     '\.$': _
+    # control characters
+    '[\x00-\x1f]': _
+    # dash at the start of a filename (causes command line ambiguity)
+    # https://unix.stackexchange.com/questions/16357/usage-of-dash-in-place-of-a-filename
+    '^-': _
+    # Bad chars to nothing
+    # starting and closing whitespace
     '\s+$': ''
     '^\s+': ''
```

Comment:
FWIW, I think trailing spaces are actually illegal on Windows! More here:
https://learn.microsoft.com/en-us/troubleshoot/windows-client/shell-experience/file-folder-name-whitespace-characters

And even more about rules for Windows filenames here:
https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file

Reply:
"Bad chars to nothing" i.e. "Bad chars [converted] to nothing", not that they're not bad to any OS. Should it say something else? I've decided that most/all URLs should be in the docs, not the config.

Reply:
That seems perfectly good! Yeah, I get the blanket usage of "bad" here. Merging shortly!